### ISSUE TITLE
energy of vaul BASE HEAL not receiving the wp bonus healing

### ISSUE THREAD

**Issue by kaspn** [WarEmu/WarBugs#20570]
the BASE heal on energy of vaul is not receiving the bonus from will power as displayed by the tool tip, and is instead only receiving the bonus from dmg.
for example the tool tip on my am with 990 wp says its heals for 1337 + 50% dmg. instead it is only healing for 1000+ 50% dmg.

**Comment by kaspn** [comment]
im aware how int scales the DAMAGE portion of the skill, wich ALSO adds an additional healing bonus. THE BASE HEAL is supposed to get a WP check.

**Comment by kaspn** [comment]
this is possibly tied in with the penetrating siphon tactic, which appears to be always active?

**Comment by Rydiak** [comment]
Back when Warrior Priest's Sigmar's Radiance had a base heal it was never affected by Willpower, and too only scaled on the lifetap portion (via strength).  So speaking consistently, this doesn't sound like a bug but rather a balance suggestion (which belongs on the forums, not here).

**Comment by kaspn** [comment]
The tooltip itself scales with willpower. Sent using the mobile mail appOn 11/23/22 at 3:15 PM, Rydiak wrote:
                
            
            
                
                    From: "Rydiak" ***@***.***>Date: November 23, 2022To: "WarEmu/WarBugs" ***@***.***>Cc: "kaspn" ***@***.***>,"Author" ***@***.***>Subject: Re: [WarEmu/WarBugs] energy of vaul BASE HEAL not receiving the wp bonus healing (Issue #20570)
                
                
Back when Warrior Priest's Sigmar's Radiance had a base heal it was never affected by Willpower, and too only scaled on the lifetap portion (via strength).  So speaking consistently, this doesn't sound like a bug but rather a balance suggestion (which belongs on the forums, not here).

**Comment by kaspn** [comment]
the base heal in the tooltip itself scales with willpower. eov is kinda unique because its gets a tactic to turn it into a full int heal with penetrating siphon which removes the base heal- however this tactic is also currently bugged and does... nothing.

**Comment by bosli42** [comment]
can confirm, EoV PSM is intelligence / magic power, both with or without tactic.
the tactic does one thing, which is change it from healcrit only to magiccrit only. this part should stay like it is, to make the skill viable at all in both specs and not just one.

**Comment by Rydiak** [comment]
Exactly. I know of no ability that checks more than one stat type for its PSM coefficient, so EoV should be no different. Especially considering the ability is an attack against another player, it makes sense for it to always check Intelligence. So yea this is probably a tooltip bug rather than an ability bug.
